### DISCUSSION TITLE
Licensing and Future of PrimeNG 16.9.1 in Angular Development

### DISCUSSION THREAD

**Discussion by secarmgotte** [primefaces/community#1069] [PrimeNG] [answered]
Hello development community!

Our team is currently immersed in a project using the Angular framework, and we're contemplating the use of PrimeNG version 16.9.1. Considering that our project is relatively small and our team is new to this technology, we are evaluating the available licensing options.

Specifically, we are keen to know if there is a possibility for PrimeNG version 16.9.1 to transition into Long-Term Support (LTS) in the future, similar to what has happened with version 16.9.2-lts. Since we estimate that the project's development and implementation will take around a year, we want to ensure we understand the licensing situation throughout that period.

Does anyone in the community have information on whether PrimeNG version 16.9.1 will remain open source over the next year, or if there are plans for this version to become LTS in the future?

We appreciate any insights or experiences you can share with us. Thank you!

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

v16.9.1 will remain as open source. We provide LTS support for the 2 versions prior to the current version. To make it clear, currently we're in v17 which means we provide LTS for v15 and v16. After the v18 comes out, LTS support for v15 will end and new LTS versions will 16 and 17.

For the further questions, please send an e-mail to contact@primetek.com.tr
